master/master.cfg

```diff
@@ -182,6 +182,24 @@ def is_important_change(change):
     return any(is_important_file(filename) for filename in change.files)
 
 
+# Avoid a build to be started between start and end time and delay such build
+# at end time
+def no_builds_between(start, end):
+    def f(builder, requests):
+        now = datetime.now()
+        if start <= now.hour < end:
```

Comment:
This condition will never be true with `start=22` and `end=2`.

Reply:
You are right. I  wrote the prototype but then left as is. I'll fix it tomorrow.

Reply:
Hopefully now the code is better to read and understand. Now it works when the time range span over midnight.